backend/src/main/java/gov/cdc/usds/simplereport/db/model/ApiUser.java

```diff
@@ -69,4 +67,29 @@ public PersonName getNameInfo() {
   public void setNameInfo(PersonName name) {
     nameInfo = name;
   }
+
+  public Set<Facility> getFacilities() {
```

Comment:
I know we talked about this in pairing so forgive me for forgetting, but could we not replace the facility getter/setter with the Lombok annotations on the `facilityAssignments` field? Get that we can't quite do that for role given the extra check for the NO_ACCESS case, but for the facilityAssignments seems like a pretty direct use case for the annotation.

Think we could also clean up the file with similar annotations on `loginEmail` and `nameInfo`

Reply:
the getter/setter Lombok annotations are not the same as these methods since they would be dealing with instances of `ApiUserFacility` instead of `Facility`. at some point we have to write logic to take a set of facilities and generate facility assignments (aka ApiUserFacility instances) for them. I could move that logic to the ApiUserService, in which case I would want to use the Lombok generated methods and I'd replace every time I use `apiUser.setFacilities(facilities)` with something like this
``` 
Set<ApiUserFacility> assignments = apiUser.getFacilityAssignments();
assignments.clear();
for (Facility facility : facilities) {
      assignments.add(new ApiUserFacility(this, facility));
}
```

so far I feel a preference for keeping this in the ApiUser class but am not super sold on this so happy to have a discussion if folks think it should be moved to the service

Reply:
I'm pushing code up in a moment to clean up the redundant getters and setters in the ApiUser class